fotoapparat/src/main/java/io/fotoapparat/hardware/Capabilities.java

```diff
@@ -73,6 +79,13 @@ public static Capabilities empty() {
         return flashModes;
     }
 
+    /**
+     * @return the sensor sensitivity capability
+     */
+    public Range sensorSensitivityCapability() {
```

Comment:
The method name is not descriptive. Would be nice if it would follow the pattern set by other methods. Something like `supportedSensorSensitivities`